hack/generate-kcp-resources.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
```

Comment:
You aren't generating all kcp resources. You are generating some kcp apis. Why the rename?

Reply:
You are not generating apis. You are generating resources of two different kinds (`API*`).

Reply:
Still not convinced changing the name that all of appstudio is using for their script that does this task is necessary

Reply:
I think those names are not appropriate and this is a file managed by us. We keep copying stuff blindly just because that's how others did it. In any case I don't want to spend much time on this. I'll rename it back.